src/styles/_catalog.scss

```diff
@@ -9,11 +11,12 @@
     display: flex;
     justify-content: center;
     align-items: center;
+
     width: 200px;
     height: 300px;
+
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
you don't need these margins, as you have   gap: 48px;

 &__card:not(:nth-child(4n)) {
   margin-right: 48px;
  }